### DISCUSSION TITLE
Is the epinio project still alive? Are there plans to put resources into it?

### DISCUSSION THREAD

**Discussion by kostecky** [epinio/epinio#2771] [General]
Hello,

I was taking a look at the frequency of code updates and it looks like there hasn't been much activity. Is this a project that plans to be continued and maintained? Just wondering if there is some insight from the community or SUSE. Thanks!

**Comment by krumware** [comment]
Hi @kostecky, this project was put on maintenance but active development is resuming. Please feel free to tag me on any issues you'd like to highlight, or ways you'd like to contribute!
